include/constants/battle_script_commands.h

```diff
@@ -168,6 +168,9 @@
 #define VARIOUS_JUMP_IF_ABSENT                  101
 #define VARIOUS_DESTROY_ABILITY_POPUP           102
 #define VARIOUS_TOTEM_BOOST                     103
+#define VARIOUS_JUMP_IF_UNDER_200				104
+#define VARIOUS_SET_SKY_DROP					105
+#define VARIOUS_CLEAR_SKY_DROP					106
```

Comment:
The values of these constants are not properly aligned with the values of the constants above and below them.

Reply:
Also, you're using tabs when you should be using empty spaces.